internal/storage/storage_handle.go

```diff
@@ -90,13 +90,16 @@ func NewStorageHandle(ctx context.Context, clientConfig StorageClientConfig) (sh
 		return
 	}
 
-	// RetryAlways causes all operations to be retried when the service returns a transient error, regardless of
-	// idempotency considerations.
 	sc.SetRetry(
```

Comment:
Adding seperate comments is little confusing. One says always retried and other says based on error.code

can you simplify and add a comment on the top saying that every operation will be retried if storageutil.shouldRetry returns true. Without storage.RetryAlways only idempotency operations will be checked for retry. Add link to the code where this logic is present.

Reply:
Done